src/Security/Member.php

```diff
@@ -382,7 +382,7 @@ public function canLogIn()
      *
      * @return bool
      */
-    public function isLockedOut()
+    protected function isLockedOut()
```

Comment:
I wouldn't mind making it public. I don't see the profit in it however. Although I agree just changing it's visibility isn't all the work that could/should be done around it.